### DISCUSSION TITLE
How to add exit function to a game

### DISCUSSION THREAD

**Discussion by JJJoker** [o3de/o3de#11017] [Help Wanted] [answered]
I tried to add an exit button to my game with scripcanvas.But I can not  find the right node，is there any suggestions？

**Comment by cgalvan** [comment, marked as answer]
There isn't a dedicated node, but I've always just used the `Execute Console Command` node with the `quit` command:

![QuitGame](https://user-images.githubusercontent.com/7519264/184428342-328afcc5-67cf-4ead-a9ad-4bc4498b7ad2.png)

**Reply by JJJoker** [reply]
Thanks for answering
